databases/foundationdb/Portfile

```diff
@@ -0,0 +1,81 @@
+long_description    \
+    FoundationDB is a scalable, fault-tolerant, ordered \
+    key-value store with full ACID transactions.  This port \
+    will install version ${version} of the FoundationDB \
+    binaries suitable for development on macOS.
+
+homepage            https://www.foundationdb.org/
+
+checksums           rmd160  2bf9a258d5db140d232fb6355fc810010824c014 \
+                    sha256  034f69d4ddf7faccd7cae2a6438d64643ac98a3a9c7b3e6ed2a03105c480df95 \
+                    size    9694306
+
+depends_lib-append  port:mono
+
+# The FoundationDB build can easily run out of memory and crash, 
+# so the recommendation is to use "ninja -j1" when building.
+use_parallel_build  no
+cmake.generator     Ninja
+
+destroot {
+    # install compiled binaries
+
+    xinstall -d -m 755 ${destroot}${prefix}/bin
+    xinstall -m 755 ${workpath}/build/packages/bin/fdbcli \
+        ${workpath}/build/packages/bin/fdbbackup \
+        ${destroot}${prefix}/bin
+    foreach x {backup_agent dr_agent fastrestore_tool fdbdr fdbrestore} {
+        ln -f -s fdbbackup ${destroot}${prefix}/bin/${x}
+    }
+    
+    xinstall -d -m 755 ${destroot}${prefix}/lib
+    xinstall -m 755 ${workpath}/build/packages/lib/libfdb_c.dylib \
+        ${destroot}${prefix}/lib
+
+    xinstall -d -m 755 ${destroot}${prefix}/libexec
+    xinstall -m 755 ${workpath}/build/packages/bin/fdbmonitor \
+        ${destroot}${prefix}/libexec
+
+    xinstall -d -m 755 ${destroot}${prefix}/sbin
+    xinstall -m 755 ${workpath}/build/packages/bin/fdbserver \
+        ${destroot}${prefix}/sbin
+
+    # install header files
+
+    xinstall -d -m 755 ${destroot}${prefix}/include/foundationdb
+    xinstall -m 755 ${worksrcpath}/bindings/c/foundationdb/fdb_c.h \
+        ${workpath}/build/bindings/c/foundationdb/fdb_c_options.g.h \
+        ${worksrcpath}/fdbclient/vexillographer/fdb.options \
+        ${destroot}${prefix}/include/foundationdb
+
+    # install sample config files and adjust paths
+
+    xinstall -d -m 755 ${destroot}${prefix}/etc/foundationdb
+    xinstall -m 755 ${worksrcpath}/packaging/osx/foundationdb.conf.new \
+        ${destroot}${prefix}/etc/foundationdb
+
+    reinplace "s|/usr/local/etc/|${prefix}/etc/|g" ${destroot}${prefix}/etc/foundationdb/foundationdb.conf.new
+    reinplace "s|/usr/local/libexec/|${prefix}/sbin/|g" ${destroot}${prefix}/etc/foundationdb/foundationdb.conf.new
+    reinplace "s|/usr/local/foundationdb/backup_agent/backup_agent|${prefix}/bin/backup_agent|g" ${destroot}${prefix}/etc/foundationdb/foundationdb.conf.new
+    reinplace "s|/usr/local/foundationdb/|${prefix}/foundationdb/|g" ${destroot}${prefix}/etc/foundationdb/foundationdb.conf.new
+}
+
+notes "
+Run these commands to install the FoundationDB Python API:
+
+pip install foundationdb==${version} && F=\$(python -c \"import fdb; print(fdb.__path__\[0\]\") && python -m compileall \$F && ln -f -s ${prefix}/lib/libfdb_c.dylib \$F/
```

Comment:
why the suggestion to install the Python API through `pip`; can MacPorts not provide a port for doing that? I think ideally we don't want to recommend using `pip` together with using MacPorts, or ...?

Reply:
The note on installing the Python API was simply to be helpful. The focus of this port is to install the FoundationDB binaries and not all the Python, Java, Go, or Ruby APIs. These would likely be provided through other Ports or manually as suggested - which would work with their Python, Java, Go, Ruby installed outside of MacPorts

Reply:
It may make sense to drop the note and leave install of any language bindings/APIs as an exercise for the user.

Reply:
I would agree with @reneeotten here.

We should remove this additional installation instructions, then provide the Python libs as either another port or a sub-port of this one.

Either way it's probably cleaner to remove the notes.

Reply:
I'm all for keeping this initial port clean :) Removed the notes and pushed an update